README.md

```diff
@@ -103,33 +103,35 @@ needed.
    hostname mptest-1.comdb2.example.com
    ```
 
-7. On one machine, create a database.
+7. On one machine, create a database - this example creates a database called _mikedb_ stored in _~/db_.
+
+   Note if still in the comdb2 repo root, the required cbd2sql binary is in place - the following commands assume this.
    ```
-   comdb2 --create --dir /home/mponomar/db mikedb
+   comdb2 --create --dir ~/db mikedb
    ```
    
 8. Configure the nodes in the cluster:
    ```
-   vi /home/mponomar/db/mikedb.lrl
+   vi ~/db/mikedb.lrl
    add
    cluster nodes mptest-1.comdb2.example.com mptest-2.comdb2.example.com
    ```
    
 9. On other nodes, copy the database over:
    ```
-   copycomdb2 mptest-1.comdb2.example.com:/home/mponomar/db/mikedb.lrl
+   copycomdb2 mptest-1.comdb2.example.com:~/db/mikedb.lrl
    ```
    
 0. On all nodes, start the database.
    ```
-   comdb2 --lrl /home/mponomar/db/mikedb.lrl mikedb
+   comdb2 --lrl ~/db/mikedb.lrl mikedb
    ```
    All nodes will say 'I AM READY.' when ready.
 
 1. On any node, start using the database.  You don't have any tables yet.  You can add them with *cdb2sql* 
    Example -
    ```sql
-   cdb2sql mikedb local 'CREATE TABLE t1 {
+   ./cdb2sql mikedb local 'CREATE TABLE t1 {
```

Comment:
One of the steps is make install and add to PATH, so ./cdb2sql should work.  Like I said, these instructions can use an update.